RHIDP-9010: AEM-related changes (#1426)
redhat-developer/red-hat-developers-documentation-rhdh: modules/customizing-the-appearance/proc-customize-rhdh-palette.adoc

      light:
        palette:
          primary:
            main: <light_primary_color>
          navigation:
            indicator: <light_indicator_color>
        pageTheme:
          default:
            backgroundColor: [<light_background_color_1>, <light_background_color_2>]
      dark:
        palette:
          primary:
            main: <dark_primary_color>
          navigation:
            indicator: <dark_indicator_color>
        pageTheme:
          default:
            backgroundColor: [<dark_background_color_1>, <dark_background_color_2>]
# ...
----

> `light:palette:primary:main`:: The main primary color for the light color palette, for example, `#ffffff` or `white`
> `light:palette:navigation:indicator`:: The color of the navigation indicator for the light color palette, which is a vertical bar that indicates the selected tab in the navigation panel, for example, `#FF0000` or `red`
> `light:pageTheme:default:backgroundColor`:: The background color for the default page theme for the light color palette, for example, `#ffffff` or `white`
> `dark:palette:primary:main`:: The main primary color for the dark color palette, for example, `#000000` or `black`
> `dark:palette:navigation:indicator`:: The color of the navigation indicator for the dark color palette, which is a vertical bar that indicates the selected tab in the navigation panel, for example, `#FF0000` or `red`
> `dark:pageTheme:default:backgroundColor`:: The background color for the default page theme for the dark color palette, for example, `#000000` or `black`

Reviewer: ```suggestion
`light|dark`:: Enter the theme name: `light` or `dark`.
`palette.primary:main`::: Enter the palette main primary color, such as `#ffffff` or `white`.
`palette.navigation:indicator`::: Enter the palette navigation indicator color, which is a vertical bar that indicates the selected tab in the navigation panel, such as `#FF0000` or `red`.
`pageTheme:default:backgroundColor`::: Enter the default page theme background color, such as `#ffffff` or `white`.
```
